pyproject.toml

```diff
@@ -1,6 +1,5 @@
 [build-system]
-requires = ["setuptools < 74"]
-build-backend = "setuptools.build_meta"
+requires = ["setuptools>=68.2,<=75.8.0", "wheel"]
```

Comment:
somehow this breaks `check-manifest`...

As this is used to build `plone.meta`, we might not need to get setuptools fixed here? 🤔

Reply:
Ah, without build-backend it fails because then we get the default build-backend, which always calls `setup.py`, which we don't have.  Okay, I have restored that line.

I think the upper bound is still useful.  Otherwise when you install plone.meta using latest buildout, the result may have the same problems.